contrib/stack/topsStack/stackSentinel.py

```diff
@@ -1010,10 +1010,12 @@ def main(iargs=None):
 
 
     #do ionosphere estimation
-    if inps.param_ion is not None:
+    if inps.param_ion is not None and os.path.isfile(inps.param_ion) == True:
```

Comment:
I think the conditions here can be a bit simpler like this, and the messages would be more helpful:
```py
if inps.param_ion is None:
    print("Ion parameter file is not specified. Ionospheric estimation will not be done.")
elif not os.path.isfile(inps.param_ion):
    print("Ion parameter file is missing. Ionospheric estimation will not be done.")
else:
    <do iono>
```

Reply:
Done. Thank you @rtburns-jpl for your suggestions.